### PR TITLE
Update conversation_tree.textproto

### PR DESCRIPTION
**Pull request by lr2kate**

### CODE REVIEW

File: conversation_tree.textproto [hunk 1]
```diff
@@ -379,8 +381,10 @@ node {
                 label: "Официальное решение"
                 url: "https://www.admin.ch/gov/de/start/dokumentation/medienmitteilungen.msg-id-87556.html"}}
         }
-   
-    
+        url {
```
Comment (skd) [inline conversation_tree.textproto:384]: This is not part of the answer. You should probably move this one line up.

Also:
python3 validate.py
should show you any errors in the script.

File: conversation_tree.textproto [hunk 2]
```diff
@@ -401,7 +405,42 @@ node {
     }
 }
 
+node {
+    name: "Изучение языка"
+    answer {
+        text: "Здесь мы собираем ссылки на предложения по изучению языков Швейцарии. "
+    }
+        url {
```
Comment (skd) [inline conversation_tree.textproto:413]: urls should be part of the answer:
```suggestion
    answer { links {
        text: "Здесь мы собираем ссылки на предложения по изучению языков Швейцарии."
        url {
            label: "Онлайн школа для детей, в том числе английски и немецкий для любого возраста"
            url: "https://t.me/UASchweiz/3122"
        }
    } }
```

Comment (skd) [inline conversation_tree.textproto:413]: urls should be part of the answer:
```suggestion
    answer { links {
        text: "Здесь мы собираем ссылки на предложения по изучению языков Швейцарии."
        url {
            label: "Онлайн школа для детей, в том числе английски и немецкий для любого возраста"
            url: "https://t.me/UASchweiz/3122"
        }
    } }

**Review by skd — CHANGES_REQUESTED** [review]
Please make sure that `python3 validate.py` does not return any errors.

**Comment by lr2kate** [comment]
I only have corp machine at the moment, can't run validate.py myself

**Comment by lr2kate** [comment]
validate.py done

**Review by skd — APPROVED** [review]

**Pull request merged**